### PR TITLE
Removed React-Native-Fingerprints-Scanner with React-native-biometrics

### PR DESCRIPTION
**Pull request by mlegall-ledger**
### 📝 Description

Replaced the legacy and unmainained lib React-Native-Fingerprints-Scanner with new and fancy React-native-biometrics

### ❓ Context

- **Impacted projects**: LLM
- **Linked resource(s)**: LIVE-4326

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

N/A

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ui/packages/crypto-icons/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "node scripts/buildReactIcons && tsc --noEmit false && node scripts/transpile",
+    "build": "pnpm run clean && node scripts/buildReactIcons && tsc --noEmit false && node scripts/transpile",
```
Comment (ofreyssinet-ledger) [inline libs/ui/packages/crypto-icons/package.json:29]: why this change here?

**Review by ofreyssinet-ledger — APPROVED** [review]

File: libs/ui/packages/crypto-icons/package.json (hunk 1 above)
Comment (mlegall-ledger) [reply libs/ui/packages/crypto-icons/package.json:29]: It was removed by error bi DX team lately, and without it you can't build locally. That change is also included in the Typescript PR

File: apps/ledger-live-mobile/package.json [hunk 2]
```diff
@@ -140,14 +140,14 @@
...
     "react-native-animatable": "^1.3.3",
+    "react-native-biometrics": "^3.0.1",
     "react-native-ble-plx": "2.0.3",
     "react-native-codegen": "^0.0.7",
     "react-native-config": "1.4.4",
     "react-native-easy-markdown": "^2.0.0",
     "react-native-extra-dimensions-android": "^1.2.5",
     "react-native-fast-crypto": "^2.2.0",
     "react-native-fast-image": "^8.5.11",
-    "react-native-fingerprint-scanner": "git+https://github.com/hieuvp/react-native-fingerprint-scanner.git#f1d136f605412d58e4de9e7e155d6f818ba24731",
```
Comment (valpinkman) [inline apps/ledger-live-mobile/package.json:150]: Thank god we remove this atrocity XD

**Review by valpinkman — APPROVED** [review]
Amazing work, thanks for unlocking everyone ;)

File: .changeset/brave-moose-bake.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": minor
```
Comment (elbywan) [inline .changeset/brave-moose-bake.md:2]: should be a patch

**Review by mcayuelas-ledger — APPROVED** [review]

**Review by Justkant — APPROVED** [review]

**Review by valpinkman — APPROVED** [review]

**Review by elbywan — APPROVED** [review]
👌

**Review by hedi-edelbloute — APPROVED** [review]

File: apps/ledger-live-mobile/package.json (hunk 2 above)
Comment (lambertkevin) [reply apps/ledger-live-mobile/package.json:150]: Just a bit sad that we traded 1 unmaintained library for another :( 
Last publish: 2 years ago 😢

Comment (valpinkman) [reply apps/ledger-live-mobile/package.json:150]: Actually not https://github.com/SelfLender/react-native-biometrics latest release is from 2 month ago
